src/components/PDFView/index.js

```diff
@@ -1,12 +1,17 @@
-import React, {memo} from 'react';
+import React, {memo, PureComponent} from 'react';
```

Comment:
There is no point to use `memo()` and `PureComponent`

Reply:
I will drop the memo. PureComponent makes more sense.